.envrc.local.template

```diff
@@ -0,0 +1,5 @@
+# If you want to make local changes to these values for testing purposes,
+# copy this template into .envrc.private, and update accordingly.
```

Comment:
Do you mean `.envrc.private` here? I initially thought I liked `.private` more than `.local`, but now I can't decide 😅.

Either way, this should match https://github.com/mbta/otp-deploy/blob/4d9fc306d8c3930612a3eef54958ef25c030ed64/.envrc#L2

Reply:
I see that `.local` is the majority of usage, so I think changing this to `.local` is appropriate.
```suggestion
# copy this template into .envrc.local, and update accordingly.
```

Reply:
lol, oops, yeah I went back and forth on the name a couple times, I changed it another place that I put .private but missed this one. I decided not to do .private because that implied to me that the values would be secrets, which this isn't (though I guess there's a remote possibility that could change eventually).